bluepysnap/edges/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""Edges and EdgePopulation objects."""
+
+from .edge_population import EdgePopulation
+from .edges import Edges
```

Comment:
These relative imports should be converted to absolute imports, for consistency with the other imports in the project and for better readability.